python/ros_interface.py

```diff
@@ -111,23 +117,32 @@ def getTrustedAvg(self, pastValue, vals):
         '''Averages the trust values, or defaults to the first term in vals if pastValue is 0 or not initialized'''
         trustedVals = self.getTrusted(pastValue, vals)
         if len(trustedVals) == 0:
-            rospy.logwarn("Values don't match with pastValue, or if pastValue is None or 0, defaulting to first sensor")
+            rospy.logwarn('No trusted values, defaulting to sensor 0')
+            rospy.loginfo('Trusted average: {}'.format(vals[0]))
             return vals[0]
         else:
-            return float(sum(trustedVals)) / len(trustedVals)
+            truAvg = float(sum(trustedVals)) / len(trustedVals)
+            rospy.loginfo('Trusted average: {} / {} = {}'.format(sum(trustedVals), len(trustedVals), truAvg))
```

Comment:
Could you also print the actual list of values? 
Eg. Trusted average of [1, 3, 4] = 8 / 3 = 2.66666
Would be nice to see the raw readings.